package.json

```diff
@@ -55,7 +55,7 @@
   ],
   "dependencies": {
     "global": "^4.4.0",
-    "video.js": "^7.17.0"
+    "video.js": "^8.0.0"
```

Comment:
There's less scope for version mismatches with imports when this is moved to `peerDependencies` and `devDependencies` in plugins. If this is a major update (because of dropping v7 support) this would be a good opportunity to change this.

Reply:
@mister-ben If this is something we want to do across the board (with other related repos), we may want to create a separate story/issue, to keep scope clean. Unless the version update is causing a mismatch now.

Reply:
Yeah, I think a separate task to update other plugins makes sense.

In this case, I think we should make this a major version of this plugin and only support Video.j 8.

To be clear, @mister-ben, you're proposing removing `video.js` from `dependencies` and adding it to both `peerDependencies` and `devDependencies`?

Reply:
Yep, `devDependencies` for working on the plugin and `peerDependencies` so installing the plugin doesn't try to install (another) Video.js. Having Video.js as a dependency of plugins can cause issues with React etc - the plugin is registered on a different `videojs` than the player is using. This has resolved these issues on a few plugins I've updated that way.

Reply:
I created a ticket on the Brightcove side to address this.